src/EnergyPlus/OutputProcessor.cc

```diff
@@ -3338,9 +3338,6 @@ namespace OutputProcessor {
         if (!ResultsFramework::resultsFramework->DYMeters.rVariablesScanned()) {
             ResultsFramework::resultsFramework->initializeMeters(EnergyMeters, ReportingFrequency::Daily);
         }
-        if (ResultsFramework::resultsFramework->DYMeters.rDataFrameEnabled()) {
-            ResultsFramework::resultsFramework->DYMeters.newRow(Month, DayOfMonth, HourOfDay, 0);
-        }
```

Comment:
I guess this fixes the meter alignment issues.

Reply:
The fixes like this and the change from Hourly to Yearly for default, yes. I didn't do a thorough testing of all combinations, but monthly works properly now. I tested with 5ZoneAirCooled.idf. I am fairly confident these fix all the issues, but if someone else could double check that would be good.